rules/DeadCode/Rector/ConstFetch/RemovePhpVersionIdCheckRector.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\DeadCode\Rector\ConstFetch;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\ConstFetch;
+use Rector\Core\Contract\Rector\ConfigurableRectorInterface;
+use Rector\Core\Rector\AbstractRector;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\ConfiguredCodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+/**
+ * @see \Rector\Tests\DeadCode\Rector\ConstFetch\RemovePhpVersionIdCheckRector\RemovePhpVersionIdCheckRectorTest
+ */
+final class RemovePhpVersionIdCheckRector extends AbstractRector implements ConfigurableRectorInterface
+{
+    /**
+     * @var string
+     */
+    public const PHP_VERSION_CONSTRAINT = '8.0';
+
+    /**
+     * @var array<string, string>
+     */
+    private const EXAMPLE_CONFIGURATION = [
+        self::PHP_VERSION_CONSTRAINT => '8.0',
```

Comment:
maybe as a second iteration: could this php version be read from `composer.json` as a config option?

Reply:
that was my thought https://github.com/rectorphp/rector/issues/6448#issuecomment-860648780 , @TomasVotruba need it to be configurable. I think that can be a fallback.